app/webpacker/styles/patterns/_text-ellipsis.scss

```diff
@@ -0,0 +1,7 @@
+.app-text-ellipsis {
+  display: -webkit-box;
+  overflow: hidden;
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp: 3;
+  line-clamp: 3;
+}
```

Comment:
@aliuk2012 My hunch is that this is not very cross-browser friendly.

Any reason we aren't doing the truncation server side?

Reply:
We had a chat about support for implimentation earlier @fofr. 

https://caniuse.com/#search=line-clamp It's supported apart from IE11.

Reply:
PLUS - you get the flexibility of truncating at differnt points based on viewport width.  🎉

Reply:
Its also worth noting that its a very small handful of providers with these long location names